[2025-09-08T17:12:13.281+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-09-08T17:12:13.288+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: cron_catchup_backfill.taskBranch backfill__2025-05-18T12:00:00+00:00 [queued]>
[2025-09-08T17:12:13.293+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: cron_catchup_backfill.taskBranch backfill__2025-05-18T12:00:00+00:00 [queued]>
[2025-09-08T17:12:13.294+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-09-08T17:12:13.305+0000] {taskinstance.py:2888} INFO - Executing <Task(BranchPythonOperator): taskBranch> on 2025-05-18 12:00:00+00:00
[2025-09-08T17:12:13.309+0000] {standard_task_runner.py:72} INFO - Started process 22666 to run task
[2025-09-08T17:12:13.312+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'cron_catchup_backfill', 'taskBranch', 'backfill__2025-05-18T12:00:00+00:00', '--job-id', '229', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/executing_cron_catchup_backfill.py', '--cfg-path', '/tmp/tmpotzsaehv']
[2025-09-08T17:12:13.313+0000] {standard_task_runner.py:105} INFO - Job 229: Subtask taskBranch
[2025-09-08T17:12:13.344+0000] {task_command.py:467} INFO - Running <TaskInstance: cron_catchup_backfill.taskBranch backfill__2025-05-18T12:00:00+00:00 [running]> on host codespaces-9bd7b3
[2025-09-08T17:12:13.422+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='loonycorn' AIRFLOW_CTX_DAG_ID='cron_catchup_backfill' AIRFLOW_CTX_TASK_ID='taskBranch' AIRFLOW_CTX_EXECUTION_DATE='2025-05-18T12:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='backfill__2025-05-18T12:00:00+00:00'
[2025-09-08T17:12:13.422+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-09-08T17:12:13.439+0000] {baseoperator.py:405} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2025-09-08T17:12:13.445+0000] {python.py:240} INFO - Done. Returned value was: taskD
[2025-09-08T17:12:13.446+0000] {branch.py:38} INFO - Branch into taskD
[2025-09-08T17:12:13.447+0000] {skipmixin.py:230} INFO - Following branch {'taskD'}
[2025-09-08T17:12:13.450+0000] {skipmixin.py:278} INFO - Skipping tasks [('taskC', -1), ('taskE', -1)]
[2025-09-08T17:12:13.473+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-09-08T17:12:13.474+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=cron_catchup_backfill, task_id=taskBranch, run_id=backfill__2025-05-18T12:00:00+00:00, execution_date=20250518T120000, start_date=20250908T171213, end_date=20250908T171213
[2025-09-08T17:12:13.524+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-09-08T17:12:13.543+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-09-08T17:12:13.550+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
